packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -100,10 +97,11 @@ class _FuchsiaLogReader extends DeviceLogReader {
     final ApplicationPackage? app = _app;
     final RegExp matchRegExp = app == null
         ? _flutterLogOutput
-        : RegExp('INFO: ${app.name}(\\.cmx)?\\(flutter\\): ');
+        : RegExp('INFO: ${app.name}(\\.cm)?\\(flutter\\): ');
     return Stream<String>.eventTransformed(
       lines,
-      (EventSink<String> output) => _FuchsiaLogSink(output, matchRegExp, startTime),
+      (EventSink<String> output) =>
+          _FuchsiaLogSink(output, matchRegExp, startTime),
```

Comment:
nit please revert this style only diff, per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

Reply:
done